pkg/adaptor/cloud/ibmcloud-powervs/provider.go

```diff
@@ -245,3 +245,11 @@ func (p *ibmcloudPowerVSProvider) getFromDHCPServer(ctx context.Context, instanc
 
 	return ip, nil
 }
+
+func (p *ibmcloudPowerVSProvider) Check() error {
```

Comment:
Shall we use StartupCheck specifically to avoid readiness or liveness probe by mistake in future?

Reply:
updated.